sdk/nodejs/storage/zMixins.ts

```diff
@@ -411,20 +441,22 @@ export class QueueEventSubscription extends appservice.EventSubscription<QueueCo
             connection: bindingConnectionKey,
         }];
 
+        args.bindings = appservice.mergeBindings(bindings, args.bindings);
+
         // Place the mapping from the well known key name to the storage account connection string in
         // the 'app settings' object.
         const appSettingsOutput = args.appSettings || pulumi.output({});
 
         // Place the mapping from the well known key name to the storage account connection string in
         // the 'app settings' object.
         const account = pulumi.all([resourceGroupName, queue.storageAccountName])
-                                .apply(([resourceGroupName, storageAccountName]) =>
-                                storage.getAccount({ resourceGroupName, name: storageAccountName }));
+                            .apply(([resourceGroupName, storageAccountName]) =>
+                            storage.getAccount({ resourceGroupName, name: storageAccountName }));

```

Comment:
same comment applies.

Reply:
fixed in d783c138f08bff57325be4e7fb98e3a6933d5b8b